library/kani/src/slice.rs

```diff
@@ -166,6 +170,10 @@ where
     since = "0.8.0",
     note = "This function may return symbolic values that don't respects the language type invariants."
 )]
-pub unsafe fn any_raw_slice<T, const MAX_SLICE_LENGTH: usize>() -> AnySlice<T, MAX_SLICE_LENGTH> {
+pub unsafe fn any_raw_slice<T, const MAX_SLICE_LENGTH: usize>() -> AnySlice<T, MAX_SLICE_LENGTH>
```

Comment:
```rust
#[cfg(not(feature = "exe_trace"))
```

Reply:
I made this into an `assert!(!cfg!(feature = "exe_trace"))`, as in `any_raw()`. I also added one to `Invariant::any()`. I reckon `AnySlice::new_raw()` doesn't need this because it isn't a public API.